src/LibraryViewExtension/LibraryViewController.cs

```diff
@@ -149,6 +152,18 @@ public void ImportLibrary()
                 dynamoViewModel.ImportLibraryCommand.Execute(null)
             ));
         }
+        /// <summary>
+        /// This function adds the events to instrumentation
```

Comment:
I don't really understand this comment, shouldn't this say something like logs the event if it matches a set of known events?

The function name doesn't make sense to me either - are there other examples of function names like this around logging and instrumentation? What are we adding instrumentation to?

Reply:
I am thinking more of `LogEventsToAnalytics` or `LogEventsToInstrumentation`. 
I couldn't find an example as analytics is global object.
thought @mjkkirschner ?

Reply:
`LogEventToInstrumentation` sounds good.